site/docs/v1/tech/guides/basic-registration-forms.adoc

```diff
@@ -0,0 +1,162 @@
+
+The "Type" radio button should have "Basic" selected. Leave it on this setting.
+
+You can toggle on the "Verify Password" switch to ask the user to enter their password twice when creating an account to ensure it is captured correctly.
+
+Leave the "Login type" radio button set on "Email".
+
+Then set the "Registration Fields" as follows:
+
+- First name: "Enabled" and "Required" on.
+- Last name: "Enabled" on and "Required" off.
+
+Set all the other fields to "Enabled" off.
+
+The settings should now look like this:
+
+image::guides/basic-registration-forms/basic-registration-settings.png[Basic Registration Settings,width=1200,role=bottom-cropped]
+
+=== Configuring Registration Verification Settings
+
+You can configure additional settings around registration verification. Scroll up to the [breadcrumb]#Registration Verification Settings# section, just above the "Registration Fields" section.
+
+Here you can specify if a registration verification email should be sent to the user. Toggling the "Verify registrations" switch on will reveal some further options.
+
+You can choose the email template to send for registration verification. Choose the "Registration Verification" template.
+
+For the "Verification Strategy" option, you can choose to send a link in the verification email to the user, or to send a short code that they can type in the registration form to verify their account.
+
+If you choose to send a short code, you must also set the "Unverified behaviour" option to "Gated". Note that the "Gated" behaviour option, and thus the short code option, is only available with a valid license key. Please visit link:/pricing[our pricing page] to review paid edition options and buy a license.
+
+Your verification settings should now look like this:
+
+image::guides/basic-registration-forms/registration-verification-settings.png[Registration Verification Settings,width=1200,role=bottom-cropped]
+
+Save all changes to the application.
+
+=== Register a User
+
+Now that the application has been configured, you can register a user. Navigate to the [breadcrumb]#Applications list# page. Click on the green button with the magnifying glass icon next to yout application to open the details popup.
+
+Scroll down to the "Registration URL" item, and copy the URL.
+
+image::guides/basic-registration-forms/registration-url.png[Register url,width=1200,role=bottom-cropped]
+
+Open the URL in a new tab, and follow the instructions to register a user.
+
+image::guides/basic-registration-forms/register-new-user.png[Register a new user,width=1200,role=bottom-cropped]
+
+The page specified by the "redirect_uri" in the "Registration URL" is the page that your users will be redirected to when they click the "Register" button. After registering, you should receive an email with a verification link. Click the link to verify the user.
```

Comment:
```suggestion
The page specified by the "redirect_uri" in the "Registration URL" is the page that your users will be redirected to when they click the "Register" button. This can be configured on the [breadcrumb]#OAuth# tab of the application. You can have more than one. 

After registering, you should receive an email with a verification link. Click the link to verify the user.
```